HygrometerApp/Sources/ViewControllers/AddViewController.swift

```diff
@@ -247,3 +257,18 @@ extension AddViewController: UITableViewDataSource {
         return cell
     }
 }
+
+//MARK: - KeyboardMonitor
+extension AddViewController {
+    /// 키보드 이벤트처리
+    fileprivate func observingKeyboardEvent() { //키보드 height를 받아서 처리
+        keyboardMonitor?.$keyboardHeight.sink { height in
+            
+            //키보드의 높이가 변할때 tempView를 띄워서 상단 터치시 dismiss처리
+            self.tempView.isHidden = height > 0 ? false : true
+ 
+        }.store(in: &subscriptions)
+    }
```

Comment:
순환참조 걸리므로 `[weak self]` 사용 부탁드립니다.

```suggestion
        keyboardMonitor?.$keyboardHeight.sink { [weak self] height in
            
            //키보드의 높이가 변할때 tempView를 띄워서 상단 터치시 dismiss처리
            self?.tempView.isHidden = height > 0 ? false : true
 
        }.store(in: &subscriptions)
    }
```